src/lib/features/scheduler/job-service.test.ts

```diff
@@ -8,6 +8,8 @@ let store: JobStore;
 const config = createTestConfig();
 beforeAll(async () => {
     db = await dbInit('job_service_serial', config.getLogger);
+    // @ts-ignore setMuteError is not part of getLogger interface
+    config.getLogger.setMuteError(true);
```

Comment:
is the setMuteError really on the LogProvider, and not the Logger?

Reply:
The provider: https://github.com/Unleash/unleash/blob/3f8336cd19fbaca8916bf0b5b671770892eba193/src/test/fixtures/no-logger.ts#L17-L19 because of how the function is defined: https://github.com/Unleash/unleash/blob/3f8336cd19fbaca8916bf0b5b671770892eba193/src/test/fixtures/no-logger.ts#L12

I don't like it much, but I also didn't want to spend much time on it